third_party/rust-on-exercism/health-statistics.rs

```diff
@@ -6,26 +6,32 @@ struct User {
 
 impl User {
     pub fn new(name: String, age: u32, weight: f32) -> Self {
+        println!("Use name {:?} age {:?} and weight{:?}", name, age, weight);
         unimplemented!()
     }
 
     pub fn name(&self) -> &str {
+        println!("Use name {:?}", self.name);
```

Comment:
I don't know if we've agreed that this is a good way to "fix" the warnings? I would have added such print statements already if this is a nice way to solve it :slightly_smiling_face: